core/src/test/scala/kafka/utils/TestInfoUtils.scala

```diff
@@ -51,7 +51,7 @@ object TestInfoUtils {
       testInfo.getDisplayName().contains("quorum=zkMigration")
     }
   }
-  final val TestWithParameterizedQuorumName = "{displayName}.quorum={0}"
+  final val TestWithParameterizedQuorumName = "{displayName}.{argumentsWithNames}"
```

Comment:
Changing this one to be aligned with https://github.com/apache/kafka/pull/14687.